java/ql/test/library-tests/dataflow/sign-analysis/SignAnalysis.expected

```diff
@@ -1,3 +1,18 @@
 | A.java:4:14:4:14 | x | strictlyNegative |
 | A.java:6:9:6:9 | x | positive |
 | A.java:7:14:7:14 | y | strictlyPositive |
+| file:///modules/java.base/java/lang/Character.class:0:0:0:0 | "1.1" | strictlyPositive |
```

Comment:
@aschackmull in C# we have a `fromSource` predicate to filter expressions that are extracted from source code, and not from binaries. Is there something similar in Java? Or should I leave these new numeric non-integer results in the expected file?

Reply:
I see the same exists on `Element` in java. Should we filter these from the results?